_posts/en/pages/2016-01-01-index.md

```diff
@@ -5,9 +5,44 @@ lang: en
 title: Bitcoin
 name: index
 permalink: /
-version: 2
+version: 3
 tags: [bitcoin, bitcoin core]
-translated: false
+
+## Strings to localize on translated versions of this page
+bitcoin-core: "Bitcoin Core"
+download-bitcoin-core: "Download Bitcoin Core"
+about1: >
+  The Bitcoin Core project provides software that helps you use Bitcoin.
+  Its flagship program is a full node that finds the valid chain
+  of blocks with the most Proof-of-Work (PoW).  The users of Bitcoin
+  Core only accept transactions on that chain, making it the Bitcoin
+  block chain that everyone else wants to use.
+
+about2: >
+  It is these users who keep Bitcoin decentralized. They
+  individually run their own Bitcoin Core full nodes, and each of
+  those full nodes separately follows the exact same rules to decide
+  which block chain is valid.
+
+about3: >
+  There's no voting or other corruptible process involved: there's
+  just individual software following identical rules to evaluate
+  identical blocks and coming to identical conclusions about which block
+  chain is valid.
```

Comment:
```suggestion
  There's no voting or other corruptible process involved: there's
  just individual software following identical rules to evaluate
  data in order to come to identical conclusions about which block
  chain is valid.
```

Reply:
What's the nit here?

Reply:
It's better (IMO) when the verbs are in the same form (to evaluate, to come).  Otherwise, since "and coming to identical conclusions about which block chain is valid" is not a full sentence, you need a comma before the "and".

I also changed "identical blocks" -> "data" because full nodes are evaluated blocks are transactions, and because they could be fed different transactions, and I guess technically different blocks by attackers.

Reply:
I don't understand:

> I also changed "identical blocks" -> "data" because full nodes are evaluated blocks are transactions, and because they could be fed different transactions, and I guess technically different blocks by attackers.

Reply:
Woops, sorry.

I also changed "identical blocks" -> "data" because full nodes evaluate blocks and transactions, and also because nodes could be fed different transactions, and I guess technically different blocks by attackers.